<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleConfig</class>
 <widget class="KTabWidget" name="ConsoleConfig">
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="generalTab">
   <attribute name="title">
    <string>General</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="3">
     <widget class="KLineEdit" name="ps2">
      <property name="whatsThis">
       <string>Continuation prompt</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="KLineEdit" name="ps1">
      <property name="whatsThis">
       <string>Main prompt</string>
      </property>
      <property name="text">
       <string>&gt;&gt;&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="labelPs2">
      <property name="text">
       <string>Continuation Prompt:</string>
      </property>
      <property name="buddy">
       <cstring>ps2</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelPs1">
      <property name="text">
       <string>Main Prompt:</string>
      </property>
      <property name="buddy">
       <cstring>ps1</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="KFontChooser" name="font">
      <property name="whatsThis">
       <string>Main display font</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="coloursTab">
   <attribute name="title">
    <string>Highlighting</string>
   </attribute>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="promptLabel">
      <property name="text">
       <string>Prompt:</string>
      </property>
      <property name="buddy">
       <cstring>promptColour</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="KColorButton" name="promptColour"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="stringLabel">
      <property name="text">
       <string>String:</string>
      </property>
      <property name="buddy">
       <cstring>stringColour</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="KColorButton" name="stringColour"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Name:</string>
      </property>
      <property name="buddy">
       <cstring>nameColour</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="KColorButton" name="nameColour"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="integerLabel">
      <property name="text">
       <string>Integer:</string>
      </property>
      <property name="buddy">
       <cstring>integerColour</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="KColorButton" name="integerColour"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="floatLabel">
      <property name="text">
       <string>Float:</string>
      </property>
      <property name="buddy">
       <cstring>floatColour</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="KColorButton" name="floatColour"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="helpLabel">
      <property name="text">
       <string>Help:</string>
      </property>
      <property name="buddy">
       <cstring>helpColour</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="KColorButton" name="helpColour"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="exceptionLabel">
      <property name="text">
       <string>Exception:</string>
      </property>
      <property name="buddy">
       <cstring>exceptionColour</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="KColorButton" name="exceptionColour"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KFontChooser</class>
   <extends>QWidget</extends>
   <header>kfontdialog.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ps1</tabstop>
  <tabstop>ps2</tabstop>
  <tabstop>promptColour</tabstop>
  <tabstop>stringColour</tabstop>
  <tabstop>nameColour</tabstop>
  <tabstop>integerColour</tabstop>
  <tabstop>floatColour</tabstop>
  <tabstop>helpColour</tabstop>
  <tabstop>exceptionColour</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
